po/pl_PL.po

```diff
@@ -0,0 +1,258 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: Gno-Menu\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2016-10-22 14:14:39-0500\n"
+"Last-Translator: Leopold L.\n"
+"Language-Team: \n"
+"Language: de_DE\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+
+#: convenience.js
+
+
+#: extension.js
+
+msgid "Gno-Menu: Search Firefox bookmarks disabled"
+msgstr "Gno-Menu: Przeszukiwanie zakładek Firefox'a wyłączone"
+
+msgid "If you want to search Firefox bookmarks, you must install the required pacakages: libgir1.2-gda-5.0 [Ubuntu] or libgda-sqlite [Fedora]"
+msgstr "Jeśli chcesz przeszukiwać zakładki Firefox'a, musisz zainstalować paczkę: libgir1.2-gda-5.0 [Ubuntu] lub libgda-sqlite [Fedora]"
+
+msgid "Gno-Menu: Search Midori bookmarks disabled"
+msgstr "Gno-Menu: MPrzeszukiwanie zakładek Midori wyłączone"
```

Comment:
Hi, i'm the one who translated it. There's mistake in line #27.
It should be "Gno-Menu: Przeszukiwanie zakładek Midori wyłączone"
Without M before "Przeszukiwanie" - my mistake. Sorry for doing it non GitHub way - i'm just a graphic designer, not a developer ;)